moulinette/interfaces/api.py

```diff
@@ -387,6 +387,15 @@ def process(self, _route, arguments={}):
             ret = self.actionsmap.process(arguments, timeout=30, route=_route)
         except MoulinetteError as e:
             raise error_to_response(e)
+        except Exception as e:
+            if isinstance(e, HTTPResponse):
+                raise e
+            import traceback
```

Comment:
That's pretty fast to load since it's a small stl module. But let's not care about that, that's fine to have it here.